### ISSUE TITLE
Display areas should be relative to the current working directory

### ISSUE THREAD

**Issue by captainsafia** [nteract/archived-desktop-app#386]
Follow-up on conversation in Slack with @rgbkrk.

Currently, if you are running the notebook in `~dev/nteract/`. You would need to run `../../../Pictures` in order to access the Pictures folder on Mac. This is because display areas currently render relative to the `static` directory, not the currently working directory.

**Comment by captainsafia** [comment]
This also applies to any commands that are sent to the kernel. So for example, something like `training = pd.read_csv('data/training.csv')` would not work. From what I looked into, this happens because `process.cwd()` points to the directory the application is being run from, `~/dev/nteract` in this case and `/Applications/Contents/nteract/blah` when we package it.

I'm looking into how the current working directory can be changed in Electron apps, and specifically how Atom does it. [This](https://github.com/electron/electron/issues/447) seemed to be the most relevant issue specifically, the last comment. I tried this on `launch` but it didn't affect `pwd`inside the notebook.

I know that in `jupyter/notebook` the `KernelManager` sets a `path` that is relative to the current working directory when it starts a kernel so maybe we have to do something like this?

Anyway, this is a bit of an incoherent brain-dump but this issue makes writing notebooks read/write commands relative to the "real" working directory no fun!
